umad/src/main/java/com/pkukielka/AccessMonitorRewriter.java

```diff
@@ -192,7 +193,7 @@ public static void addLockToCurrentThread(Object o) {
     }
 
     public static void removeLockFromCurrentThread(Object o) {
-        locks.get().remove((Integer)System.identityHashCode(o));
+        locks.get().remove((Integer) System.identityHashCode(o));
```

Comment:
add ` // We need to cast to `Integer` because otherwise it would be treated as index` as well